tests/unit/scene/test_ipynb_conversion.py

```diff
@@ -0,0 +1,118 @@
+"""Unit tests for the conversion from and to ipynb."""
+
+from pytest_mock import MockerFixture
+import pytest_check as check
+import json
+
+from opencodeblocks.scene.from_ipynb_conversion import ipynb_to_ipyg, is_title
+from opencodeblocks.scene.ipynb_conversion_constants import BLOCK_TYPE_TO_NAME
+
+
+class TestIpynbConversion:
+
+    """Testing function_parsing functions"""
```

Comment:
Wrong name, also look at clipboard test for naming conventions, they are different for tests.

```suggestion
    """Conversion from .ipynb"""
```